assets/scripts/segments/buildings.js

```diff
@@ -253,11 +254,10 @@ export function drawBuilding (ctx, variant, floors, position, totalWidth, totalH
       multiplier, dpi)
 
     // middle floors
-    const randomGenerator = new RandomGenerator()
-    randomGenerator.seed = 0
+    const randomGenerator = seedrandom(generateRandSeed())
 
     for (let i = 1; i < floors; i++) {
-      const variant = (building.variantsCount === 0) ? 0 : Math.floor(randomGenerator.rand() * building.variantsCount) + 1
+      const variant = (building.variantsCount === 0) ? 0 : Math.floor(randomGenerator() * building.variantsCount) + 1
```

Comment:
🤔 part of me wonders if we should explore using UUIDs, but that's definitely out of scope here